source/api_documentation/call_ingestion_api/index.rst

```diff
@@ -0,0 +1,321 @@
+For example, if you pass an **advertiser_campaign_id_from_network** that you do not have access to, the following error will be returned.
+
+**Response (403 Forbidden):**
+
+.. code-block:: json
+
+    {
+      "errors": {
+        "class": "UnauthorizedAdvertiser",
+        "invalid_data": "You do not have access to this advertiser"
+      }
+    }
+-----
+
+**Campaign Configuration Related Errors**
+
+In order to fully utilize the Call Ingestion API, there are some configuration requirements for the campaign that the call is being submitted under.  Here's a list of those requirements:
+  * Campaigns must be setup with a campaign type of **External**
+  * Campaigns need to be configured with transciption enabled
+  * Campaigns need to have Signal AI enabled at at least one Signal configured
+
+If any of these settings are misconfigured you'll see error message similar to the examples below.  
+*Please contact the Invoca support team at questions@invoca.com for setup assistance.* 
+
+**Response (403 Forbidden):**
+
+.. code-block:: json
+
+    {
+      "errors": {
+        "class": "call.advertiser_campaign_id_from_network",
+        "invalid_data": "campaign must be for external calls only"
+      }
+    }
+
+**Response (403 Forbidden):**
+
+.. code-block:: json
+
+    {
+      "errors": {
+        "class": "call.advertiser_campaign_id_from_network",
+        "invalid_data": "campaign must have transcription enabled"
+      }
+    }
+
+Supported Recording Formats
+---------------------------
+
+The default call recording format on the Invoca platform is 16-bit PCM encoded `WAV <https://en.wikipedia.org/wiki/WAV>`_ files with an 8 kHz sample rate. 
+The Call Ingestion API supports `WAV <https://en.wikipedia.org/wiki/WAV>`_  and `MP3 <https://en.wikipedia.org/wiki/MP3>`_ file formats.  However, the Invoca Audio Processing system will upsample or downsample accordingly into our default call recording format.
+
+All call recordings are required to be in dual-channel or stereo format.  The call recording of an inbound call on the Invoca platform has the caller channel on channel 0 and the agent audio on channel 1. 
+For all calls submitted via the Call Ingsetion API, we will normalize the channels to match a the Invoca call record channel layout.
+
+The **call_direction** field will determine how the recording is normalized:
+
+    `inbound` The audio processing system will assume that the call recording matches the Invoca default with the caller channel on channel 0 and the agent channel on channel 1.
+
+    `outbound` The audio processing system will assume that the call recording is the opposite of the Invoca dafault.  The audio procesing system will normalize the call recording by swapping the channels.
+
+
+If the Invoca Audio Processing system finds any call recording format problems then a message will be sent via email notifying you of the issue.  Please see the **Call Processing Error Notifications** section for more details.
```

Comment:
how is the email address specified? is it all emails on the network account? Can we clarify who/how this is configured

Reply:
For beta we are going to be sending these emails to the network CSM

Reply:
Thanks @nburwell and @sasha-romanova updates pushed.